automated/linux/alsa-smoke/alsa-smoke.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+# shellcheck disable=SC1091
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+RESULT_FILE="${OUTPUT}/result.txt"
+export RESULT_FILE
+REFERENCE_PATH="/dev/snd"
+
```

Comment:
SKIP_INSTALL should be set her to "False"

Reply:
Fixed.